### PR TITLE
logs for master components

### PR DESCRIPTION
**Pull request by ArtfulCoder**
Logs from kube-controller-manager, kube-apiserver and kube-scheduler pods are written to corresponding /var/log files

### CODE REVIEW

File: cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest [hunk 1]
```diff
@@ -47,6 +47,8 @@
 {% endif -%}
 {% endif -%}
 
+{% set params = "--master=127.0.0.1:8080" + " " + machines + " " + cluster_name + " " + minion_regexp + " " + cloud_provider  + " " + sync_nodes + " " + cloud_config + " " + pillar['log_level'] -%}
```
Comment (dchen1107) [inline cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest]: why --master=127.0.0.1:8080 required for logging kube-controller-manager?

**Comment by dchen1107** [comment]
Can you rebase first, then run e2e test for this?

**Comment by ArtfulCoder** [comment]
rebased and ran e2e.
Looks good.

File: cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest (hunk 1 above)
Comment (ArtfulCoder) [inline cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest]: we always had --master=127.0.0.1:8080. check linke 62 of the checked in version (on left side).

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**